src/ObjectSerializerInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Serializer;
+
+interface ObjectSerializerInterface
```

Comment:
Need a description here. When to use the interface and not `SerializerInterface` would be the main question users will ask.